.github/workflows/configure.cmd

```diff
@@ -32,14 +32,14 @@ del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\cmake_install.cmake"
 del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
 del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"
 
-cmake                                                ^
-  -G"Visual Studio 16 2019"                          ^
-  -H"%KRATOS_SOURCE%"                                ^
-  -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"             ^
-  -DBOOST_ROOT="%TEMP%\boost"                        ^
-  -DINSTALL_RUNKRATOS=OFF                            ^
-  -DCMAKE_CXX_FLAGS="/Od /we4661 /we4804 /WX"        ^
-  -DFORCE_LOCAL_ZLIB_COMPILATION=ON                  ^
+cmake                                                 ^
+  -G"Visual Studio 16 2019"                           ^
+  -H"%KRATOS_SOURCE%"                                 ^
+  -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"              ^
+  -DBOOST_ROOT="%TEMP%\boost"                         ^
+  -DINSTALL_RUNKRATOS=OFF                             ^
+  -DCMAKE_CXX_FLAGS="/Od /we4661 /we4804 /WX /wd4996" ^
```

Comment:
this disables deprecation warnings
This is required as we turn all warnings into errors
=> same as we do in the linux CI

I am wondering how this was working before, we have a few other deprecation warnings in the code!